From 83cdab8b2c6ea0fc0860f8444d083353b47f1d5c Mon Sep 17 00:00:00 2001
From: Tom Rini <trini@konsulko.com>
Date: Tue, 11 Jul 2023 11:20:44 -0400
Subject: [PATCH] Prepare v2023.07.02

Take the v2023.07 tag, which reported to be v2023.07-rc6 still, and
change this to be v2023.07.02 (as I made a v2023.07.01 incorrectly).

Signed-off-by: Tom Rini <trini@konsulko.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 73b71b61c7a..e7703616c50 100644
--- a/Makefile
+++ b/Makefile
@@ -2,8 +2,8 @@
 
 VERSION = 2023
 PATCHLEVEL = 07
-SUBLEVEL =
-EXTRAVERSION = -rc6
+SUBLEVEL = 02
+EXTRAVERSION =
 NAME =
 
 # *DOCUMENTATION*
